packages/@jsii/go-runtime/jsii-runtime-go/helpers.go

```diff
@@ -2,23 +2,45 @@ package jsii
 
 import "time"
 
+type Type interface {
+	bool | string | float64 | time.Time
+}
+
+func V[T Type](v T) *T {
```

Comment:
This is missing a doc-string. Also I'm not a huge fan of the non-descriptive name here...

Reply:
Yes, I thought that would a contentious point!

What do you think, `Ptr` (because it returns a pointer value), `Value` (because it's a `jsii.Value`), or something else?